src/RepoAuditor/Plugins/ScientificSoftware/Requirements/Citation.py

```diff
@@ -59,5 +59,6 @@ def __init__(self) -> None:
                 https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files
                 """
             ),
-            dynamic_arg_name="unrequired",
+            enabled_by_default=False,  # Don't require citation file to be present.
```

Comment:
I think that this should be enabled by default now that the requirement has moved to a new module.